assets/src/edit-story/app/story/storyProvider.js

```diff
@@ -66,6 +68,7 @@ function StoryProvider( { storyId, children } ) {
 	const setPropertiesById = useSetPropertiesById( { currentPageIndex, pages, setPages } );
 
 	useLoadStory( { storyId, pages, setPages, setTitle, setPostStatus, setPostAuthor, setSlug, setLink, setCurrentPageIndex, clearSelection } );
+	useLoadFonts( { fonts, setFonts } );
```

Comment:
In the future, I think we should either lazy load this request or just load fonts that are in current use. 
Thoughts @barklund